### PR TITLE
#971 Billing: BigQuery - New daily file upload

### PR DESCRIPTION
**Pull request by trevor2718**
# Description

Aggregating dev-billing data, generating report csv and uploading it to S3. 
Updating dev-billing table with the report from S3.
#971 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Send messages with Postman using `/v2/notifications/sms` endpoint and keep track of the number of messages sent, grouping by the service, template, and billing code. 
- When `create_nightly_billing_for_day(process_day)` runs every day at 0015 ET the numbers you sent should match the data retrieved. You can verify by checking the file with the given date in the `env-notifications-daily-billing-stats` s3 bucket, or the `dev-billing` table in BigQuery should also contain the data in those files.

- [x] All unit tests pass

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by trevor2718** [comment]
Working on passing tests.

**Comment by cris-oddball** [comment]
Please remove the word 'fixes' from the description.

File: app/celery/reporting_tasks.py [hunk 1]
```diff
@@ -142,3 +143,27 @@ def generate_daily_notification_status_csv_report(process_day_string):
...
+    ]
+    writer.writerow(header)
+    writer.writerows((process_day,) + row for row in transit_data)
+
+    csv_key = f'{process_day_string}.csv'
+    client = boto3.client('s3', endpoint_url=current_app.config['AWS_S3_ENDPOINT_URL'])
+    client.put_object(Body=buff.getvalue(), Bucket=current_app.config['DAILY_STATS_BUCKET_NAME'], Key=csv_key)
+    buff.close()
+
+    current_app.logger.info(f"generate-daily-billing-sms-per-use-case-csv-report complete: "
```
Comment (cris-oddball) [inline app/celery/reporting_tasks.py:168]: For any logs written, please convert to %s string notation (might be worth just going through this entire file for this type of logging).

Comment (trevor2718) [reply app/celery/reporting_tasks.py]: @cris-oddball Just updated. Please check it out

Comment (EvanParish) [reply app/celery/reporting_tasks.py]: It's not the right format. It should replace the variables with "%s". It should look something like this:
`logger.info("complete: %s", var1)`

**Review by cris-oddball — DISMISSED** [review]
Nice job on the TODOs with ticket numbers. FYI, if possible later, once we get the sender/sender_id sorted, if we can go back to a list comprehension to write the rows, that will be more efficient and faster than the current for loop.

**Review by cris-oddball — APPROVED** [review]

**Pull request merged**